frontend/styles/globalStyles.css

```diff
@@ -42,15 +61,57 @@ a {
   color: black;
 }
 
-a {
-  text-decoration: none;
-  color: black;
-}
-
 .j-link {
-  color: #4391C9;
+  color: #4391c9;
 }
 
 .j-link:hover {
   text-decoration: underline;
 }
+
+em {
+  font-style: italic;
+}
+
+strong {
+  font-weight: 700;
+}
+
+input,
+label,
+select,
+button,
+textarea {
+  margin: 0;
+  border: 0;
+  padding: 0;
+  display: inline-block;
+  vertical-align: middle;
+  white-space: normal;
+  background: none;
+  line-height: 1;
+  background-color: white;
+}
+
+input:focus {
+  outline: 0;
+}
+
+input,
+textarea {
+  -webkit-box-sizing: content-box;
+  -moz-box-sizing: content-box;
+  box-sizing: content-box;
```

Comment:
How come `content-box` for all input types except the ones below?

Reply:
I just brought over what we had before. Changed them all to be unified ✅